locale/en_US/app.json

```diff
@@ -0,0 +1,54 @@
+{
+  "Become a Maker": {
+    "message": "Become a Maker",
+    "description": "Call to action to become a Maker using Webmaker's app"
+  },
+  "Strips away complexity": {
+    "message": "Webmaker takes the best part of the web — creating — and strips away the complexity. ",
+    "description": "Explanation about how Webmaker makes creating on the we less complex for Makers"
+  },
+  "Build the Perfect App": {
+    "message": "Build the Perfect App",
+    "description": "Header describing what Webmaker's app can be used for - building your own apps"
+  },
+  "Exactly what you need": {
+    "message": "Create exactly what you need: A website for your business, a chat app for your friends, a lesson plan for your students. ",
```

Comment:
Same here. Is there a reason for this? I'm afraid we are going to miss this in the translation process.